dist/notify.js

```diff
@@ -202,4 +207,6 @@
 
   exports['default'] = Notify;
 
+  Object.defineProperty(exports, '__esModule', { value: true });
```

Comment:
Yep, please also update the dev dependency in `package.json` here to use the latest version of `jspm@beta` beta.